zerver/lib/events.py

```diff
@@ -312,6 +312,7 @@ def fetch_initial_state_data(
         state["is_guest"] = settings_user.is_guest
         state["user_id"] = settings_user.id
         state["enter_sends"] = settings_user.enter_sends
+        state["final_escape_narrow"] = settings_user.final_escape_narrow
```

Comment:
Let's call this field `default_view` -- I think that better expresses the idea given that we're planning to have this also control what loads when you open the app.

Reply:
Okay.